### PR TITLE
bgp,test: Properly wait for FRR container to be ready

### PR DESCRIPTION
**Pull request by YutaroHayakawa**
Please see commit message for more details.

```release-note
bgp,test: Properly wait for FRR container to be ready
```

### CODE REVIEW

File: test/k8s/bgp.go [hunk 1]
```diff
@@ -66,15 +66,14 @@ var _ = SkipDescribeIf(
 			BeforeAll(func() {
 				frr = applyFRRTemplate(kubectl, ni)
 				kubectl.ApplyDefault(frr).ExpectSuccess("Unable to apply rendered template %s", frr)
+				kubectl.WaitForSinglePod("kube-system", "frr", 30*time.Second)
```
Comment (viktor-kurchenko) [inline test/k8s/bgp.go]: You ignore the error check here because the next line will give us the guarantee, right?

**Review by viktor-kurchenko — APPROVED** [review]

**Review by dylandreimerink — APPROVED** [review]

File: test/k8s/bgp.go (hunk 1 above)
Comment (YutaroHayakawa) [reply test/k8s/bgp.go]: No, I just forgot to handle it 😅 Fixed in the latest push.

**Comment by YutaroHayakawa** [comment]
Need to rebase to get https://github.com/cilium/cilium/pull/24422

**Comment by YutaroHayakawa** [comment]
Cilium Runtime: I triggered this CI too early before finishing to build the images. Rerunning.

**Comment by YutaroHayakawa** [comment]
All unexecuted tests are skipped because this PR only contains the changes under `test/`. Ready to merge.

**Pull request merged**